src/hooks/__data__/mock-data.ts

```diff
@@ -514,3 +515,166 @@ export const mockLimitRange: LimitRange = {
     ],
   },
 };
+
+export const mockedServiceAccount: ServiceAccountKind = {
+  kind: 'ServiceAccount',
+  apiVersion: 'v1',
+  metadata: {
+    name: 'build-pipeline-c7814',
+    namespace: 'rh-ee-rgalvao-tenant',
+    uid: '1e50e721-a93d-41d4-8f3a-8da999c67d1c',
+    resourceVersion: '1842281725',
+    creationTimestamp: '2025-04-17T12:55:25Z',
+    ownerReferences: [
+      {
+        apiVersion: 'appstudio.redhat.com/v1alpha1',
+        kind: 'Component',
+        name: 'c7814',
+        uid: '6a9b6b2b-77d1-4815-974a-dfd6b4caebe9',
+      },
+    ],
+  },
+  secrets: [
+    {
+      name: 'build-pipeline-c7814-dockercfg-bksxm',
+    },
+    {
+      name: 'c7814-image-push',
+    },
+    {
+      name: 'testing-new-secret',
+    },
+    {
+      name: 'aha-new-test',
+    },
+  ],
+  imagePullSecrets: [
+    {
+      name: 'build-pipeline-c7814-dockercfg-bksxm',
+    },
+    {
+      name: 'testing-new-secret',
+    },
+    {
+      name: 'aha-new-test',
+    },
+  ],
+};
+
+export const mockedSecret1: SecretKind = {
+  metadata: {
+    name: 'a036c-image-pull',
+    namespace: 'rh-ee-rgalvao-tenant',
+    uid: 'bbb8afad-4043-4eba-93c3-bb51331e88cd',
+    resourceVersion: '1796664926',
+    creationTimestamp: '2025-04-14T09:07:41Z',
+    deletionTimestamp: '2025-04-17T12:55:26Z',
+    labels: { 'appstudio.redhat.com/internal': 'true' },
+    ownerReferences: [
+      {
+        apiVersion: 'appstudio.redhat.com/v1alpha1',
+        kind: 'ImageRepository',
+        name: 'a036c',
+        uid: 'c91d4817-d9a7-44ff-abf7-0e0ad6f036bb',
+      },
+    ],
+  },
+  data: {
+    '.dockerconfigjson': 'eyJhdXR',
+  },
+  type: 'kubernetes.io/dockerconfigjson',
+  apiVersion: 'v1',
+  kind: 'Secret',
+};
+export const mockedSecret2: SecretKind = {
```

Comment:
As usual, we do not mock all data directly. For some similar ones, we mock one and edit when needed.
Is there existing mock data would use directly, like mockSecret in Secrets/__data__/mock-secret.ts ?
then {...mockSecert, type:..} to do some edit.

Reply:
I'll simplify the mocked data!